File: tests/go/cases/case_182_deferred_sink_gate_09.go
package escape_tests

var retainedCase182 = []map[string]string{}

func Case182DeferredSinkGate09(input string) string {
	raw := input
	if raw == "" {
		raw = "sample"
	}
	payload := map[string]string{
		"task": "deferred_sink_gate_09",
		"entity": "stress",
		"stage": "evaluation",
		"input": raw,
		"checksum": "deferred_sink_gate_09:" + raw,
	}
	if len(raw) > 0 && raw[0] == 'x' {
		// ESCAPE: conditional path retention.
		retainedCase182 = append(retainedCase182, payload)
	}
	return "ok"
}

File: tests/nodejs/cases/case_182_deferred_sink_gate_09.js
/**
 * Task case 182: deferred_sink_gate_09 false-positive/false-negative stress pattern.
 */

const retainedCache = new Map();
const retainedAudit = [];
const retainedHandlers = [];

function case182DeferredSinkGate09(input) {
  const taskName = 'deferred_sink_gate_09';
  const raw = input || 'sample';
  const payload = {
    task: taskName,
    entity: 'stress',
    stage: 'evaluation',
    input: raw,
    checksum: `${taskName}:${raw.length}`,
  };
  if (raw.startsWith('x')) {
    // ESCAPE: conditional sink only on selected path.
    retainedAudit.push(payload);
  }
  return 'ok';
}

module.exports = {
  case182DeferredSinkGate09,
};
